src/bit_array.cr

```diff
@@ -125,4 +222,20 @@ struct BitArray
   private def malloc_size
     (@size / 32.0).ceil.to_i
   end
+
+  # FIXME: this was copied from Array, we should deduplicate implementations
```

Comment:
Let's move it as a `:nodoc:` in `Indexable` (and also deduplicate `String#range_to_index_and_size` ;-) )